src/main/java/net/dv8tion/jda/api/entities/GuildWelcomeScreen.java

```diff
@@ -0,0 +1,123 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.api.entities;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class GuildWelcomeScreen
+{
+    private final String description;
+    private final List<GuildWelcomeScreen.Channel> channels;
+
+    public GuildWelcomeScreen(String description, List<GuildWelcomeScreen.Channel> channels)
+    {
+        this.description = description;
+        this.channels = channels;
+    }
+
+    /**
+     * The server description that is shown in the Welcome screen.
+     *
+     * @return The server description that is shown in the Welcome screen
+     */
+    @Nullable
+    public String getDescription()
+    {
+        return description;
+    }
+
+    /**
+     * The channels that are shown in the Welcome screen.
+     *
+     * @return Possibly-empty, unmodifiable list of the channels that are shown in the Welcome screen
+     */
+    @Nonnull
+    public List<GuildWelcomeScreen.Channel> getChannels()
+    {
+        return channels;
+    }
+
+    /**
+     * POJO for the recommended channels information provided by a Welcome screen.
+     * <br>Recommended channels are shown in the Welcome screen after joining a server.
+     *
+     * @see GuildWelcomeScreen#getChannels()
+     */
+    public static class Channel implements ISnowflake {
+        private final long id;
+        private final String description;
+        private final String emoteId;
+        private final String emojiName;
+
+        public Channel(long id, String description, String emoteId, String emojiName)
+        {
+            this.id = id;
+            this.description = description;
+            this.emoteId = emoteId;
+            this.emojiName = emojiName;
+        }
+
+        /**
+         * The id of this recommended channel.
+         *
+         * @return The id of this recommended channel
+         */
+        @Override
+        public long getIdLong()
+        {
+            return id;
+        }
+
+        /**
+         * The description of this recommended channel that is shown in the Welcome screen.
+         *
+         * @return The description of this recommended channel
+         */
+        @Nonnull
+        public String getDescription()
+        {
+            return description;
+        }
+
+        /**
+         * The id of the emote that is used for this recommended channel.
+         * <br><b>This will return {@code null} if the emoji is a unicode emoji.</b>
+         *
+         * @return The id of the emote that is used for this recommended channel or {@code null}
+         */
+        @Nullable
+        public String getEmoteId()
```

Comment:
I'm not sure whether I should wrap this into an `Emoji` object instead. The class constructor accepts the `animated` field which we can't know, while a recommended channel's emote can be animated.

Reply:
We can't tell by the emote's id via something like `a_` prefix or whatever?

Reply:
no, the object only contains the emote id and name, not the hash